clean_enoa/call_inegi.py

```diff
@@ -0,0 +1,56 @@
+import requests
+import os
+import json
+
+
+def output(content, index):
+    per = content["Series"][0]["OBSERVATIONS"][index]["TIME_PERIOD"]
+    val = content["Series"][0]["OBSERVATIONS"][index]["OBS_VALUE"]
+    print(per, val)
+
+
+def get_content(url):
+    response = requests.get(url)
+    if response.status_code == 200:
+        content = json.loads(response.content)
+    return content
+
+
+def desc_grupo(content, grupo):
+    ocupado_2021 = int(content["Series"][0]["OBSERVATIONS"][0]["OBS_VALUE"])
+    ocupado_2020 = int(content["Series"][0]["OBSERVATIONS"][4]["OBS_VALUE"])
```

Comment:
Pareciera que el 4 es un número mágico. ¿pordríamos encapsularlo de alguna manera?

Además, en el enunciado de abajo no mencionas 2020 ni 2021, solo mencionas año anterior. ¿Los nombres `ocupado_2020` y `ocupado_2021` son adecuados? (los números en los nombres de las variables despiden cierto olor)

Reply:
Me estaba imaginando algo así:

```suggestion
    ocupado = {
        2021 : int(content["Series"][0]["OBSERVATIONS"][0]["OBS_VALUE"]),
        2020 : int(content["Series"][0]["OBSERVATIONS"][4]["OBS_VALUE"])
        }
```

Entonces puedes acceder a los datos de la siguiente manera: `ocupado[2020]` y `ocupado[2021]`. Así, el año no es parte del nombre, y lo puedes usar como el valor de una variable, podrías iterar sobre el año, etc.
```
primer_ano = 2020
ultimo_ano = 2021
ocupado[primer_ano]
ocupado[ultimo_ano]
```

Reply:
Esta propuesta está muy bonita, muchas gracias. :heart_eyes_cat:

Reply:
Listo @evaristor ya quedó:
```python
indicator = {
        "last_quarter": int(content["Series"][0]["OBSERVATIONS"][0]["OBS_VALUE"]),
        "a_year_ago": int(content["Series"][0]["OBSERVATIONS"][4]["OBS_VALUE"]),
    }
 
```
Y lo utilicé así `{indicator['last_quarter']}`.